SIAB/spillage/api.py

```
@@ -754,6 +772,43 @@ def _nzeta_analysis(folder, count_thr = 1e-1, itype = 0):
 
 class TestAPI(unittest.TestCase):
 
+    def test_coef_gen(self):
+
+        rcut = 3.0
+        ecut = 10.0
+        lmax = 3
+        coefs = _coef_gen(rcut, ecut, lmax)
+        self.assertEqual(len(coefs), 1) # always only one atomtype
+        self.assertEqual(len(coefs[0]), lmax + 1) # lmax + 1 orbitals
+        for l in range(lmax + 1):
```

Ruff

Ambiguous variable name: `l`

(E741)